cmd/krel/cmd/ff_test.go

```diff
@@ -120,13 +120,13 @@ func TestFfSuccess(t *testing.T) {
 	defer s.cleanup(t)
 
 	ffo := s.getFfOptions()
-	ffo.branch = pseudoReleaseBranch
+	ffo.Branch = pseudoReleaseBranch
 
 	ro := s.getRootOptions()
 	ro.nomock = true
```

Comment:
We do not need the root options any more, but we have to set the nomock for the ffo:
```suggestion
	ro := s.getRootOptions()
	ffo.NoMock = true
```